docs/reference/run.md

```diff
@@ -94,6 +94,20 @@ You can reattach to a detached container with `docker`
 [*attach*](/reference/commandline/attach). If you choose to run a
 container in the detached mode, then you cannot use the `--rm` option.
 
+When starting a container in detached mode, one needs to understand that, by
```

Comment:
@moxiegirl thanks for your inputs. Just a suggestion. Let me know what you think. Otherwise what you commented makes perfect sense. I added stuff assuming we would also have readers who have very less knowledge on the topic.

> This attempt fails because the root process (service x start) returns and the detached container stops as designed.

This succeeds in starting the `nginx` service inside the container but fails the detached container paradigm in that, the root process (`service nginx start`) returns and the detached container stops as designed. Hence, rendering the `nginx` service thus started of no use.